src/api/oauth.rs

```diff
@@ -0,0 +1,13 @@
+use crate::client::LichessApi;
+use crate::error::Result;
+use crate::model::oauth::*;
+
+impl LichessApi<reqwest::Client> {
+    pub async fn test_tokens(&self, request: impl Into<test::PostRequest>) -> Result<TestResults> {
+        self.get_single_model(request.into()).await
+    }
+
+    pub async fn revoke_token(&self) -> Result<()> {
+        self.get_empty(revoke::DeleteRequest::new()).await
```

Comment:
This might be better handled as setting up an expectation for a 204 response. That way if there's other responses that occur (e.g. unauthorized etc.), these might have some context about what was expected to happen and what happened instead. What do you think?